src/components/selectboxes/SelectBoxes.js

```diff
@@ -205,14 +205,17 @@ export default class SelectBoxesComponent extends RadioComponent {
     const maxCount = this.component.validate.maxSelectedCount;
     const isValid = this.isValid(data, dirty);
 
-    if (maxCount || minCount) {
+    if ((maxCount || minCount) && this.visible) {
```

Comment:
It might be better to use shouldSkipValidation method from parent class instead of just visibility check..

Reply:
changed